packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -304,8 +355,9 @@ export class DataStores implements IDisposable {
     }
 
     public async getDataStore(id: string, wait: boolean): Promise<FluidDataStoreContext> {
-        const context = await this.contexts.getBoundOrRemoted(id, wait);
+        const internalId = this.aliasMap.get(id) ?? id;
```

Comment:
I'm not 100% sure that's the right choice to consult this.aliasMap.get here. It's definitely right choice for ContainerRuntime.getRootDataStore() flow. The other flows (which seems to converge around only one public entry-point - ContainerRuntime.resolveHandle()) - more questionable. I think we have to do it for now (for back compat), but long term I think we need to have a discussion here.